migrations/8_token_acc_key.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE fungible_tokens ADD COLUMN fbt_pubkey bytea NOT NULL;
```

Comment:
Why not drop and recreate it?

Reply:
yeah, we need to drop old table because on devnet it is already filled and such migration may fail